public/components/dragDropEditor.tsx

```diff
@@ -59,7 +60,7 @@ export const DragDropEditor = ({ value, http, renderToast }) => {
   const onSave = async () => {
     try {
       const body = JSON.stringify({ metadashboard });
-      const response = await http.put('/api/dashboards/metadashboard/edit', {
+      const response = await http.put(`${API_PREFIX}/metadashboard`, {
```

Comment:
A better name would be `menu` because we only add data for the bap menu and not something else.